tests/e2e/import.test.js

```diff
@@ -0,0 +1,199 @@
+      const value = await page.evaluate(el => el.textContent, labelErrorElement);
+
+      expect(value).toBe('The passwords gotta match, smh!');
+    });
+
+    test('Fails on password shorter than 12 characters', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(secrets.seedphrase);
+
+      const submitImport = await utils.getButtonWithIndex(page, 0);
+      await submitImport.click();
+
+      const [passwordInput, confirmPasswordInput] = await utils.getInputs(page);
+      await passwordInput.type('123');
+      await confirmPasswordInput.type('123');
+
+      const submitPassword = await utils.getButtonWithIndex(page, 0);
+      await submitPassword.click();
+
+      await page.waitForSelector(passwordErrorLabel);
+      const labelErrorElement = await page.$(passwordErrorLabel)
+      const value = await page.evaluate(el => el.textContent, labelErrorElement);
+
+      expect(value).toBe('The minimum is 12 characters, smh!');
+    });
+
+    test('Correctly imports', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(secrets.seedphrase);
+
+      const submitImport = await utils.getButtonWithIndex(page, 0);
+      await submitImport.click();
+
+      const [passwordInput, confirmPasswordInput] = await utils.getInputs(page, true);
+      await passwordInput.type(secrets.password);
+      await confirmPasswordInput.type(secrets.password);
+
+      const submitPassword = await utils.getButtonWithIndex(page, 0);
+      await submitPassword.click();
+
+      await page.goto(chromeData.popupUrl);
+
+      const popupPasswordInput = await utils.getInputWithIndex(page, 0, true);
+      await popupPasswordInput.type(secrets.password);
+
+      const unlockPlugButton = await utils.getButtonWithIndex(page, 0);
+      await unlockPlugButton.click();
+
+      const [plugBanner] = await utils.getXPathElements(page, 'span', 'Alpha Release', true);
+      const value = await page.evaluate(el => el.textContent, plugBanner);
+
+      expect(value).toMatch(/Plug/i);
+    });
+  });
+
+  describe('Create', () => {
+    beforeEach(async () => {
+      const createButton = await utils.getButtonWithIndex(page, 1);
+      await createButton.click();
+    });
+
+    test('Fails on missmatched passwords', async () => {
+      const [passwordInput, confirmPasswordInput] = await utils.getInputs(page);
+      await passwordInput.type('TestPassword123');
+      await confirmPasswordInput.type('MissMatchedPassword');
+
+      const submitPasswordButton = await utils.getButtonWithIndex(page, 0);
+      await submitPasswordButton.click();
+
+      await page.waitForSelector(passwordErrorLabel);
+      const labelErrorElement = await page.$(passwordErrorLabel)
+      const value = await page.evaluate(el => el.textContent, labelErrorElement);
+
+      expect(value).toBe('The passwords gotta match, smh!');
```

Comment:
seems kinda the same as import right? do you see any way to abstract this or is it impossibru?

Reply:
Yes! we technically could abstract this, however I decided not to do so since it's precisely testing the import/create flows. However I did create an `importAndUnlock` util function that can be used in other tests (i.e Send) so we don't have to worry about importing/creating accounts.